pkgs/games/spring/springlobby.nix

```diff
@@ -54,10 +54,6 @@ stdenv.mkDerivation rec {
   patches = [
     ./revert_58b423e.patch # Allows springLobby to continue using system installed spring until #707 is fixed
     ./fix-certs.patch
-    (fetchpatch {
```

Comment:
Please remove the unused fetchpatch from inputs.

Reply:
Done.